build.gradle

```diff
@@ -118,6 +118,7 @@ sourceSets {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
remove

Reply:
I needed `mavenLocal()` to be able to load the patched version `org.jabref:afterburner.fx:2.0.0-SNAPSHOT`.

Not required any more.